### PR TITLE
pkg/controller: Clean up template rendering code

### PR DESCRIPTION
**Pull request by sgreene570**
**- What I did**
Cleaned up some code and comment blocks.
**- How to verify it**

**- Description for the changelog**

Clean up code and comment blocks in pkg/controller.

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
this is great.  

/lgtm

File: pkg/controller/template/render.go [hunk 1]
```diff
@@ -366,24 +366,19 @@ func etcdPeerCertDNSNames(cfg RenderConfig) (interface{}, error) {
...
 	switch cfg.Platform {
-	case platformAWS:
-		return platformAWS, nil
-	case platformAzure:
-		return platformAzure, nil
-	case platformOpenstack:
-		return platformOpenstack, nil
-	case platformVSphere:
-		return platformVSphere, nil
+	case platformAWS, platformAzure, platformOpenstack, platformVSphere:
```
Comment (runcom) [inline pkg/controller/template/render.go]: loving this

**Comment by kikisdeliveryservice** [comment]
ok it was rebased

/lgtm

**Pull request merged**